frontend/src/screens/landing/widgets/tutorials/tutorials.js

```diff
@@ -0,0 +1,179 @@
+    fontWeight: 400,
+  },
+  link: {
+    textDecoration: 'none',
+  },
+  gridRow: {
+    display: 'grid',
+    gridTemplateColumns: '12vw calc(70vw - 12vw)',
+    width: '70vw',
+    height: 'calc(100vh - 223px - 1.2vw - 2em - 3.15vh - 3.15vh)',
+  },
+  controlsContainer: {
+    width: '12vw',
+    borderRight: '1px solid darkgrey',
+    overflow: 'auto',
+    paddingRight: '10px',
+    boxSizing: 'border-box',
+  },
+  viewer: {
+    width: 'calc(70vw - 12vw)',
+    position: 'relative',
+  },
+  title: {
+    fontSize: '2em',
+    margin: '3.15vh 0',
+  },
+  text: {
+    fontSize: '1.2vw',
+  },
+  textItem: {
+    fontSize: '1vw',
+    cursor: 'pointer',
+    display: 'block',
+    textAlign: 'center',
+    borderBottom: '1px solid #333',
+    width: '100%',
+    paddingBottom: '3px',
+  },
+  fadeLoaderFull: {
+    position: 'absolute',
+    paddingLeft: 'calc((100vw - 40px) / 2)',
+    paddingTop: 'calc(22vh)',
+  },
+  fadeLoader: {
+    position: 'absolute',
+    paddingLeft: 'calc(70vw - 41vw)',
+    paddingTop: 'calc(22vh)',
+  },
+  space: {
+    paddingBottom: '8px',
+  },
+};
+
+class Tutorials extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      idVideo: '',
+    };
+  }
+
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+  };
+
+  loadStart = () => {
+    this.setState({ loading: true });
+  };
+
+  loadEnd = () => {
+    this.setState({ loading: false });
+  };
+
+  handleClickVideo = idVideo => {
+    this.setState({ idVideo });
+    if (this.state.idVideo !== idVideo) this.loadStart();
+  };
+
+  renderLoading = () => {
+    if (!this.state.loading) return null;
+    const showControls = this.state.idVideo;
+    const classLoading = showControls
+      ? styles.fadeLoader
+      : styles.fadeLoaderFull;
+    if (this.state.idVideo !== '') {
+      return (
+        <div className={this.props.classes.loading}>
+          <FadeLoader
+            style={classLoading}
+            color="#424242"
+            size="16px"
+            margin="4px"
+          />
+        </div>
+      );
+    }
+  };
+
+  renderControls = () => {
+    const { classes } = this.props;
+    return (
+      <div className={classes.controlsContainer}>
+        <ExpansionPanel>
+          <ExpansionPanelSummary expandIcon={<Icon>expand_more</Icon>}>
+            <Typography className={classes.text}>Processing History</Typography>
+          </ExpansionPanelSummary>
+          <ExpansionPanelDetails className={classes.space}>
+            <Typography className={classes.textItem}>
+              <span onClick={() => this.handleClickVideo('YLt5ouupWbk')}>
+                QA
+              </span>
+            </Typography>
+          </ExpansionPanelDetails>
+          <ExpansionPanelDetails>
+            <Typography className={classes.textItem}>
+              <span onClick={() => this.handleClickVideo('FT1T0OjvLO8')}>
```

Comment:
Make the `onClick` outside so it's possible to click on the button instead of just the text:

```
          <div onClick={() => this.handleClickVideo('YLt5ouupWbk')}>
            <ExpansionPanelDetails className={classes.space}>
              <Typography className={classes.textItem}>QA</Typography>
            </ExpansionPanelDetails>
          </div>
```